torch/testing/_asserts.py

```diff
@@ -766,19 +783,21 @@ def assert_close(
     Raises:
         UsageError: If a :class:`torch.Tensor` can't be constructed from an array-or-scalar-like.
         UsageError: If only :attr:`rtol` or :attr:`atol` is specified.
-        AssertionError: If corresponding array-likes have different types.
+        AssertionError: If corresponding tensor-or-scalar-likes do not share a common superclass.
```

Comment:
"If neither actual or expected are Python numbers, share the same type, or one is not a (direct or indirect) subclass of the other"?